tools/pot/openvino/tools/pot/configs/hardware/gna.json

```diff
@@ -37,34 +52,39 @@
             "type": "Convolution",
             "quantization": {
                 "activations": "q16_a",
-                "weights": "q16_w"
+                "weights": "q16_w",
+                "outputs": "q32_o"
```

Comment:
@elilobanova to comment on what should be a default precision for outputs of Convolution and MatMul. I mean what precision is in the output of these primitives in the plugin?

Reply:
@AlexKoff88 It depends on fusing. Without activation these layers has I32 output.